### PR TITLE
Targeting!!!

### PR DESCRIPTION
**Pull request by dpannasch**
## Motivation / Description

Targeting!!!!

### CODE REVIEW

File: docs_source/🛠 Tools/targeting.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+### Definitions
+
+[block:parameters]
+{
+  "data": {
+    "h-0": "Dimension",
+    "h-1": "Definition",
+    "0-0": "Country",
+    "0-1": "The storefront a customer is currently using on supported SDK versions\\*, or their geolocation otherwise.  \n  \n\\*Available on iOS SDK Version x and Android SDK version y",
```
Comment (dpannasch) [inline docs_source/🛠 Tools/targeting.md:149]: @joshdholtz Mind updating this when we know those SDK versions? Feel free to rephrase it too w/ the full list of SDKs and versions.

File: docs_source/🛠 Tools/targeting.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
+---
+title: Targeting
+slug: targeting
+excerpt: Define audiences to target with unique Offerings
+hidden: true
+---
+Targeting allows you to create rules for serving distinct audiences their own Offering. Instead of having a single Current Offering that all customers receive, you can instead create a cascading sequence of rules to deliver different Offerings to each audience that you define.
+
+This allows you to create paywall experiences that are tailored to each of your audiences so you can make an effective pitch for your product and maximize lifetime value.
```
Comment (aboedo) [inline docs_source/🛠 Tools/targeting.md:9]: since we talked about audiences twice in this section, should we consider making it a part of the terminology?

File: docs_source/🛠 Tools/targeting.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+| Live             | The Targeting Rules that are actively being used to determine which customers which receive Offerings, as determined by their conditions, assessed in order from top to bottom.                                        |
+| Inactive         | The Targeting Rules that are not actively being used. These may be drafts, rules you previously used,  rules you intend to set live in the future, etc.                                                                |
+
+## How Targeting works
+
+Before you setup any Targeting Rules, if you use Offerings, here's how they're returned in your app:
+
+1. RevenueCat is initialized
+2. Offerings are fetched
+3. Your list of Offerings is returned, along with the identifier of your **Current Offering**
```
Comment (aboedo) [inline docs_source/🛠 Tools/targeting.md:29]: This isn't inaccurate from a REST API perspective, but I'd phrase it more like "One of them is marked as your Current Offering" or something. 

Like, as an app developer, you really just get an Offering that you access through "current", rather than an identifier for it (https://github.com/RevenueCat/purchases-ios/blob/main/Sources/Purchasing/Offerings.swift#L41)

File: docs_source/🛠 Tools/targeting.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+
+As long as your app is setup to display the Current Offering on your paywall, then you can change that Offering at any time from our Dashboard, or run an Experiment to serve two different Offerings to specific audiences.
+
+Once you setup Targeting Rules, you unlock an additional level of customization, because the **Current Offering** that gets returned for each customer will be based on the Rule they qualify for. For example:
+
+1. RevenueCat is initialized
+2. Offerings are fetched
+3. Your list of Offerings is returned, along with the identifier of the Offering for the first rule that the customer matched as the **Current Offering** for that customer
```
Comment (aboedo) [inline docs_source/🛠 Tools/targeting.md:37]: same thought here, maybe just referring to it as "along with the current offering" and bypassing the identifier part of it does the trick already

File: docs_source/🛠 Tools/targeting.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+
+> 📘 
+> 
+> Think of your "default Offering" just like your "Current Offering" if you have no Targeting Rules setup. It's the Offering that will get served if no other rules apply.
+
+When determining which (if any) Targeting Rule a customer matches, we'll assess them from top to bottom as you've ordered them in the Dashboard.
+
+## Creating & Ordering Targeting Rules
+
+First, navigate to "Targeting" in the "Monetization tools" section of your Project Settings. Then click on "Create a New Rule" to begin.
```
Comment (aboedo) [inline docs_source/🛠 Tools/targeting.md:48]: should we have some form of an intro here explaining what ordering means in this context?

File: docs_source/🛠 Tools/targeting.md (hunk 2 above)
Comment (dpannasch) [reply docs_source/🛠 Tools/targeting.md:9]: At first I didn't want to since it's not an "entity" like many of the other terms, but after describing this feature in a few more places I do think it's important to standardize this -- I'll add it

File: docs_source/🛠 Tools/targeting.md (hunk 3 above)
Comment (dpannasch) [reply docs_source/🛠 Tools/targeting.md:29]: Yeah I like that phrasing better, I'll incorporate it

File: docs_source/🛠 Tools/targeting.md (hunk 5 above)
Comment (dpannasch) [reply docs_source/🛠 Tools/targeting.md]: Yeah now that Creating and Ordering are two independent actions, I actually just split them up into separate sections

File: docs_source/🛠 Tools/targeting.md (hunk 4 above)
Comment (dpannasch) [reply docs_source/🛠 Tools/targeting.md:32]: 10-4, updated

**Comment by dpannasch** [comment]
NOTE: It looks like we will update current Offering to default Offering on the Dashboard (proposal [here](https://revenuecat.slack.com/archives/CFU1JG9U6/p1700595848075709)). When we do that, this doc (and others) will get a round of changes.
